scripts/check-format.sh

```diff
@@ -33,9 +33,9 @@ unformatted_files=""
 badly_named_files=""
 for file in $(find "$@" -name "*.h" -or -name "*.hpp" -or -name "*.cpp" -or -name "*.c"); do
   # Workaround for https://bugs.llvm.org/show_bug.cgi?id=39216
-  d=$(cat "$file" | clang-format-7 -style=file --assume-filename "${file%.*}".cpp | diff "$file" -)
```

Comment:
Slightly unintended consequence of no longer running the Open Enclave dependencies playbooks.